common/include/pcl/common/impl/transforms.hpp

```diff
@@ -220,7 +220,7 @@ struct Transformer<double>
 template <typename PointT, typename Scalar> void
 transformPointCloud (const pcl::PointCloud<PointT> &cloud_in,
                      pcl::PointCloud<PointT> &cloud_out,
-                     const Eigen::Transform<Scalar, 3, Eigen::Affine> &transform,
+                     const Eigen::Matrix<Scalar, 4, 4> &transform,
```

Comment:
Perhaps add these in the public API as well?

Reply:
[They already are in the public API](https://github.com/PointCloudLibrary/pcl/pull/4398/files?file-filters%5B%5D=.h#diff-7cf1df3a44eadef2f837e7c11fb06a00L219), only now they have a direct implementation.

Reply:
Ah, I see what happened here.